OneBusAway/OBAApplicationDelegate.m

```diff
@@ -219,6 +219,21 @@ + (OBADeepLinkRouter*)setupDeepLinkRouterWithModelDAO:(OBAModelDAO*)modelDAO app
 }
 
 - (BOOL)application:(UIApplication *)application continueUserActivity:(NSUserActivity *)userActivity restorationHandler:(void (^)(NSArray *))restorationHandler {
+    NSInteger regionID = [userActivity.userInfo[[OBAHandoff regionIDKey]] integerValue];
```

Comment:
Please use property syntax for `regionIDKey`. Xcode doesn't handle the code completion for it, but it works fine:

```
NSInteger regionID = [userActivity.userInfo[OBAHandoff.regionIDKey] integerValue];
```